cmd/ai-proxy/conf/erda-platforms.yml

```diff
@@ -0,0 +1,12 @@
+platforms:
+  - name: erda.cloud
+    openapi: https://openapi.erda.cloud
+    description: "erda.cloud 中心"
+
+  - name: erda.daily.terminus.io
+    openapi: https://openapi.daily.terminus.io
+    description: "集成环境"
+
+  - name: one.seagts.terminus.io
```

Comment:
Exposing these on the public network is not a good choice.

Reply:
removed this